packages/database/src/migrations/20210607020237-AddIndicatorsForAuFlutracking-modifies-data.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+import { deleteObject, generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  id: generateId(),
+  code: 'FluTracker_LGA_Percent_First_Nations_ILI',
+  builder: 'analyticArithmetic',
+  config: {
+    formula:
+      'firstExistingValue(FWV_LGA_ILI_Percentage_prior_March_2021, FWV_LGA_ILI_Percentage_Till_Current)',
+    parameters: [
+      {
+        builder: 'analyticArithmetic',
+        code: 'FWV_LGA_ILI_Percentage_prior_March_2021',
+        config: {
+          formula: 'FluTracking_NationalFNILI',
+          aggregation: ['FINAL_EACH_DAY'],
+        },
+      },
+      {
+        builder: 'analyticArithmetic',
+        code: 'FWV_LGA_ILI_Percentage_Till_Current',
+        config: {
+          formula:
+            '(FWV_LGA_004a_Total_By_Country >= 0 and FWV_LGA_003a_Total_By_Country > 0) ? (FWV_LGA_004a_Total_By_Country / FWV_LGA_003a_Total_By_Country * 100) : 0',
```

Comment:
```suggestion
            'FWV_LGA_003a_Total_By_Country === 0 ? 0 : FWV_LGA_004a_Total_By_Country / FWV_LGA_003a_Total_By_Country',
```

Couple of things going on here:
1. No need to check if the numerator is greater than 0. We do it for the denominator as div by 0 can cause unexpected behaviour, but if for whatever reason the numerator is tracked as a negative number we should just let it be.
2. We generally treat percentages in our system as decimals, ie. `50%` = `0.5`, so no need to multiple by 100

Reply:
That is a good question @rohan-bes ! But I have noticed the result of `FWV_LGA_ILI_Percentage_prior_March_2021` have a few greater than 1 (100%), and the result of `FWV_LGA_ILI_Percentage_Till_Current` is very small, like 0.0006. But yeah that should be a question to data stewards. I will ask on Tuesday.

Reply:
@rohan-bes For point one, checking numerator value is to see if undefined, do you think indicator will not throw error if undefined? We can trust the data input though.

Reply:
Hmm that's a good point, I think it's very likely that sometimes we will end up with `undefined / undefined`

While I think the best solution would be for us to migrate the `divide()` function from the report-server to the `expression-parser` to allow for division of undefined (should just return undefined) unfortunately there's no good place in the code for those functions to sit...

I think for now we can just use a more verbose formula, eg:
`FWV_LGA_004a_Total_By_Country === 'undefined' or FWV_LGA_003a_Total_By_Country === 'undefined' ? 'undefined' : FWV_LGA_003a_Total_By_Country === 0 ? 0 :  (FWV_LGA_004a_Total_By_Country / FWV_LGA_003a_Total_By_Country * 100))`